docs/content/preview/yugabyte-platform/_index.md

```diff
@@ -24,30 +29,161 @@ You can access YugabyteDB Anywhere via an Internet browser that has been support
+      <a href="administer-yugabyte-platform/" title="Administer">
+        <div class="box border">
+          <div class="other-content">
+            <div class="heading">Administer</div>
+            <div class="detail-copy">Back up and restore the server, as well as configure authentication for the server login.</div>
+          </div>
+        </div>
+      </a>
+    </div>
+    <div class="col-12 col-md-6 col-lg-4">
+      <a href="upgrade/" title="Upgrade">
+        <div class="box border">
+          <div class="other-content">
+            <div class="heading">Upgrade</div>
+            <div class="detail-copy">Upgrade to a newer version of YugabyteDB Anywhere.</div>
+          </div>
+        </div>
+      </a>
+    </div>
+-->
+  </div>
+</div>
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="configure-yugabyte-platform/configure/aws/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/quick_start/create_cluster.png" aria-hidden="true" />
-        <div class="title">Configure</div>
+<div class="three-box-row">
+  <div class="row">
+    <h2 class="col-12">Additional resources</h2>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border two-side-">
+        <div class="other-content">
+          <div class="heading">Build applications</div>
+          <div class="detail-copy">Start coding in your favorite programming language using examples.</div>
+          <a class="text-link" href="../develop/build-apps/" title="Get started">Get started</a>
+        </div>
       </div>
-      <div class="body">
-        Configuring YugabyteDB Anywhere for various cloud providers
+    </div>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border two-side-">
+        <div class="other-content">
+          <div class="heading">Yugabyte University</div>
+          <div class="detail-copy">Take free courses and workshops to learn YugabyteDB, YSQL, and YCQL.</div>
+          <ul>
+            <li><a class="text-link" target="_blank" href="https://university.yugabyte.com/collections/builder-workshop" title="Course 2" target="_blank" rel="noopener">Developer workshops</a></li>
+            <li><a class="text-link" target="_blank" href="https://university.yugabyte.com/courses/ysql-exercises-simple-queries" title="Course 3" target="_blank" rel="noopener">YSQL exercises</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border two-side-">
+        <div class="other-content">
+          <div class="heading">Support</div>
+          <div class="detail-copy">Ask questions, request assistance from our team, and participate in our journey.</div>
+          <ul>
+            <li><a class="text-link" target="_blank" href="https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431" title="Contact Support">Contact Support</a></li>
```

Comment:
Opens to form with YB Managed enetered

Reply:
Oh

Reply:
Do you know if they call YBA "Yugabyte Enterprise" in the support combo box?

Reply:
That must be for Anywhere, sure